drivers/flash/spi_nor.c

```diff
@@ -540,6 +540,45 @@ static int spi_nor_wrsr(const struct device *dev,
 	return ret;
 }
 
+#define MX25R_HIGH_PERF_MODE BIT(1)
+
+#ifdef CONFIG_SPI_NOR_MX_HIGH_PERF_MODE
+static int mx_specific_enable_high_perf_mode(const struct device *dev, uint8_t jedec_id[])
+{
+	int sr, ret;
+	uint8_t data[3];
+
+	/* Only supported on Macronix MX25R Ultra Low Power series. */
+	if (jedec_id[0] != 0xc2 || jedec_id[1] != 0x28) {
```

Comment:
I am not insisting on defining identifiers for the numbers, but at least give some comment on what they mean (vendor code, device series).

Reply:
Not needed anymore, since the new device tree binding is only for that manufacturer and series.